docs/datamodel/scalars/numeric.rst

```diff
@@ -93,15 +123,13 @@ from :eql:type:`str` and :eql:type:`json`.
 
     All of the following types can be explicitly cast into decimal:
     :eql:type:`str`, :eql:type:`json`, :eql:type:`int16`,
-    :eql:type:`int32`, :eql:type:`int64`, :eql:type:`float32`, and
-    :eql:type:`float64`.
+    :eql:type:`int32`, :eql:type:`int64`, :eql:type:`float32`,
+    :eql:type:`float64`, and :eql:type:`bigint`.
 
-    A decimal type has it's own literal:
+    A decimal literal is a float literal followed by 'n':
```

Comment:
I would say this need a clarification. That either decimal dot is required or `e` ?

Reply:
This is literally clarified by an example after the `:`.  I added a `e` example as well.